persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/GeneMapper.xml

```diff
@@ -15,6 +15,17 @@
         </if>
     </sql>
 
+    <sql id="selectRefGene">
```

Comment:
We will replace current \"select\" with \"selectRefGene\" in a future PR?

Reply:
I think we should, it would not be hurt just leave it there for now.

Reply:
yes.

Reply:
no, all queries in GeneMapper will still use "select" whereas CopyNumberSegmentMapper and MutationMapper will import and use "selectRefGene".

Reply:
but there is a `join` missing in this case, no?

Reply:
and a `where reference_genome_gene.REFERENCE_GENOME_ID = ...`

Reply:
@pieterlukasse , "selectRefGene" is not for GeneMapper but for CopyNumberSegmentMapper and MutationMapper in a future PR. The join clause will be supplied by query objects in either CopyNumberSegmentMapper or MutationMapper.

Reply:
@khzhu thanks for the explanation. It would make more sense (and simplify the review process) if `selectRefGene` is added together with the code where it is used. So not in this PR but in the subsequent PR.

Reply:
Looks like this is the last item. Can we resolve this? @khzhu: are you OK to move this into next PR?